timesketch/frontend-ng/src/components/Explore/AggregateDialog.vue

```diff
@@ -0,0 +1,537 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-card class="mx-auto">
+    <v-card-title>
+      <span class="headline">Event Data Analytics</span>
+      <v-spacer></v-spacer>
+      <v-btn icon @click="clearAndCancel">
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+    </v-card-title>
+    <v-card-subtitle>
+      <span class="text-h6">Field:&nbsp;<span style="font-family: monospace">{{ eventKey }}</span> | Value: &nbsp;<span style="font-family: monospace">{{ eventValue }}</span></span>
+    </v-card-subtitle>
+    <v-card-text>
+      <v-container fluid>
+        <v-row justify="center">
+          <v-col>
+            <v-card height="450px" :loading="!statsReady">
```

Comment:
I took a stab at the general statistics card, since it looks a bit lost visually compared to the other elements and I have a hard time to read/understand the data. 

What do you think about this approach:
![general_stats](https://user-images.githubusercontent.com/99879757/226875401-ed7b1edb-0ad0-41ea-a751-376d11632f5b.png)

If you like it, here is the code I used. Feel free to adjust it to your preferences.
```
<v-col>
  <v-card height="145px" :loading="!statsReady">
    <v-card-title>
      Sketch statistics
    </v-card-title>
    <v-simple-table dense v-if="statsReady" class="px-2 mt-n4">
      <tbody>
        <tr>
          <td width="360px">Total number of events</td>
          <td><strong>{{ this.docCount }}</strong></td>
        </tr>
        <tr>
          <td width="360px">First event</td>
          <td><strong>{{ this.fieldDateTimeMinimum }}</strong></td>
        </tr>
        <tr>
          <td width="360px">Last event</td>
          <td><strong>{{ this.fieldDateTimeMaximum }}</strong></td>
        </tr>
      </tbody>
    </v-simple-table>
  </v-card>
  <v-card height="145px" :loading="!statsReady" class="mt-1">
    <v-card-title>
      Field statistics
    </v-card-title>
    <v-simple-table dense v-if="statsReady" class="px-2 mt-n4">
      <tbody>
        <tr>
          <td width="360px">Field name</td>
          <td><span style="font-family: monospace">{{ this.eventKey }}</span></td>
        </tr>
        <tr>
          <td width="360px">Total number of events</td>
          <td><strong>{{ this.fieldValueCount }}</strong></td>
        </tr>
        <tr>
          <td width="360px">Number of unique values</td>
          <td><strong>{{ this.fieldCardinality }}</strong></td>
        </tr>
      </tbody>
    </v-simple-table>
  </v-card>
  <v-card height="180px" :loading="!statsReady" class="mt-1">
    <v-card-title>
      Value statistics
    </v-card-title>
    <v-simple-table dense v-if="statsReady" class="px-2 mt-n4">
      <tbody>
        <tr>
          <td width="360px">Field value</td>
          <td><span style="font-family: monospace">{{ this.eventValue }}</span></td>
        </tr>
        <tr>
          <td width="360px">Total number of events</td>
          <td><strong>{{ this.valueEventCount }}</strong></td>
        </tr>
        <tr>
          <td width="360px">First event</td>
          <td><strong>{{ this.valueDateTimeMinimum }}</strong></td>
        </tr>
        <tr>
          <td width="360px">Last event</td>
          <td><strong>{{ this.valueDateTimeMaximum }}</strong></td>
        </tr>
      </tbody>
    </v-simple-table>
  </v-card>
</v-col>
```

Reply:
thanks, i was having a think today and wasn't quite sure if these values needed an additional visualisation or formatting but agree this looks better.

Reply:
Sketch statistics is a bit malplaced IMO. I would not expect to get that information when I click on a key/value.

Reply:
I would argue that the general information on how many events a sketch has and which time window the sketch covers provides a good reference point for the analyst to set a context on the statistic values provided for field and value.

Reply:
I made the suggested changes with some tweaks on column width.  

The idea for having the sketch statistics is as described by @jkppr - to get a reference of the selected value time frame relative to the entire sketch.  Another option would be to use those values in another visualisation or two?

Let me know how you would like to make additional changes.

Reply:
Since I don't see this as blocking, I would keep the sketch statistics for now as context information. Let's collect some user feedback when this PR is in and then revisit it in the second iteration.

Reply:
@sydp small UI nit: At least on my system the Sketch statistics and Field statistics v-card are 1px too small in height resulting in the table overlaying the bottom border line.

![general_stats](https://user-images.githubusercontent.com/99879757/227532218-56de3473-a99a-412f-ac70-c309783f7d01.png)

Reply:
I made some adjustments to reveal the border lines